pages/getting_started/web_console.rst

```diff
@@ -1,11 +1,15 @@
+.. _connect:
```

Comment:
in future, use little longer anchors - having short is nice to write but having better names will make it easy to remember where this links to.